### DISCUSSION TITLE
Updating Error in Pop-OS

### DISCUSSION THREAD

**Discussion by Ayu-code-max** [rustdesk/rustdesk#7300] [Q&A]
I don't have any technical knowledge of linux terminal.

This problem arised recently while I was trying to update my Pop-OS(22.04):
![JJY3h](https://github.com/rustdesk/rustdesk/assets/141343377/a063734b-bcd7-4bfa-9cb2-a2288166575b)
How do I solve this?

**Comment by xlionjuan** [comment]
Wrong place to ask .....

Please fix your VSCode repository

Or I'm more recommend to install it though Canonical Snapcraft
